swatch-producer-azure/src/main/java/com/redhat/swatch/azure/kafka/streams/BillableUsageAggregationStreamProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.azure.kafka.streams;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import lombok.Data;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Data
+@ApplicationScoped
+public class BillableUsageAggregationStreamProperties {
+
+  @ConfigProperty(name = "KSTREAM_BILLABLE_USAGE_STORE")
+  private String billableUsageStoreName;
+
+  @ConfigProperty(name = "KSTREAM_BILLABLE_USAGE_SUPPRESS_STORE")
+  private String billableUsageSuppressStoreName;
+
+  @ConfigProperty(name = "BILLABLE_USAGE_TOPIC")
+  private String billableUsageTopicName;
+
+  @ConfigProperty(name = "BILLABLE_USAGE_HOURLY_AGGREGATE_TOPIC")
+  private String billableUsageHourlyAggregateTopicName;
+
+  @ConfigProperty(name = "KSTREAM_BILLABLE_USAGE_AGGREGATION_WINDOW_SECONDS")
+  private int windowSeconds;
+
+  @ConfigProperty(name = "KSTREAM_BILLABLE_USAGE_AGGREGATION_GRACE_SECONDS")
+  private int graceSeconds;
```

Comment:
Please use `Duration` as the type here instead of `int` (and rename to windowDuration, etc). (Will need to add `s` suffix I think to the values).